src/main/java/nl/ictm2a4/javagame/screens/GameScreen.java

```diff
@@ -217,4 +220,24 @@ private void registerAchievements() {
         new FirstDoorOpened();
         new LastLevelAchieved();
     }
+
+    public void setCurrentUser(User user){
+        currentUser = user;
+    }
+
+    public String getApiToken(){
+        if(currentUser != null){
+            return currentUser.token;
+        } else {
+            return null;
+        }
+    }
+
+    public int getCurrentUserId(){
```

Comment:
Is deze functie nodig? Heeft User geen getID() functie?

Reply:
Jazeker, dit is de huidige ingelogde user, niet het ID van een specifieke user.

Reply:
Ja maar je slaat toch ook al de huidige user op? En die kun je vervoglens ook ophalen, of zie ik dit verkeerd?

Reply:
Ja dat klopt, maar dit werkt ook. Als je liever getCurrentUser hebt moet je dat zeggen

Reply:
Nou het is meer zo dat je nu dubbele code gebruikt. Want ik zag dat je op sommige plekken getCurrentUserID == 0 gebruikt om te checken of iemand ingelogd is? Dat schept, iig voor mij, verwarring. Dus zou liever hebben dat je overal hetzelfde gebruikt. En dan lijkt mij getCurrentUser() handiger toch?